src/main/resources/templates/components/navbar.peb

```diff
@@ -5,9 +5,11 @@
         <li><h1>Koil.</h1></li>
     </ul>
     <ul class="is-hidden-mobile" data-test="desktop-navbar">
-        {% if global.isAdmin %}
-            <li><a href="/admin" data-test="navbar-admin-link">Admin Panel</a></li>
-        {% endif %}
+        <li>
+            <a href="/dashboard/user-settings" data-test="user-image">
+                <img class="profile-image" src="/dashboard/user-settings/image" alt="User profile image">
```

Comment:
If on mobile we are using the alt as the account handle- should we be doing the same on the profile-image here?